README.org

```diff
@@ -195,10 +243,17 @@ Then launch it with =emacs --with-profile rational=!
 [[http://makeapullrequest.com][https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square]]
 [[https://github.com/bbatsov/emacs-lisp-style-guide][https://img.shields.io/badge/elisp-style%20guide-purple.svg?style=flat-square]]
 
-This is a community-run modular Emacs configuration, for which we appreciate feedback in the form of issues and pull requests. Feel free to open an issue prior to opening a pull request if you're not certain your idea is in the spirit of the [[https://github.com/SystemCrafters/rational-emacs/blob/master/README.org#Principles][Principles]].
+This is a community-run modular Emacs configuration, for which we
+appreciate feedback in the form of issues and pull requests. Feel free
+to open an issue prior to opening a pull request if you're not certain
+your idea is in the spirit of the [[https://github.com/SystemCrafters/rational-emacs/blob/master/README.org#Principles][Principles]].
 
-If you enjoy crafting your computing experience, join the [[https://systemcrafters.net/][SystemCrafters]] community!
+If you enjoy crafting your computing experience, join the
+[[https://systemcrafters.net/][SystemCrafters]] community!
 
 * License
 
-This code is licensed under the MIT License.  Why?  So you can copy the code from this configuration!
+This code is licensed under the MIT License. Why? So you can copy the
+code from this configuration!
```

Comment:
If we're going to stick with line wrapping in this file, I'd recommend turning on `auto-fill-mode` as a file-local variable.  That would help anyone who edits it make sure that the file is wrapped correctly.

Can you add this to the end of the file too?

```org
# Local Variables:
# fill-column: 80
# eval: (auto-fill-mode 1)
# End:
```

Reply:
Check!